test/ngMock/angular-mocksSpec.js

```diff
@@ -351,6 +351,18 @@ describe('ngMock', function() {
     }));
 
 
+    it('should allow you to NOT specify the delay time', inject(function($interval) {
+      var counter = 0;
+      $interval(function() {counter++;});
+      $interval(function() {counter++;}, 0);
+
+      $interval.flush(1000);
+      expect(counter).toBe(2000);
+      $interval.flush(1000);
+      expect(counter).toBe(4000);
+    }));
```

Comment:
We need a test with other, non-zero-delay timeouts, to verify that the relative orders are correct.